tests/assets/recipes/invalid/fixed_loop_key.error

```diff
@@ -0,0 +1 @@
+Loop Control Key must have templated values from the
```

Comment:
```suggestion
Loop Control Key must have templated values from the [???]
```

Reply:
This file is used to check the error message output from the `fixed_loop_key.yaml` recipe file. As such this only needs to partially match the error message to confirm my test works as expected.